From e1a6b368b656e922e719c37044051d01ced90730 Mon Sep 17 00:00:00 2001
From: Tim Paine <3105306+timkpaine@users.noreply.github.com>
Date: Wed, 5 Apr 2023 11:09:32 -0400
Subject: [PATCH] pybind_cmake

---
 dist/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dist/CMakeLists.txt b/dist/CMakeLists.txt
index aafd8e649..94ed422d4 100644
--- a/dist/CMakeLists.txt
+++ b/dist/CMakeLists.txt
@@ -324,8 +324,8 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
             find_package(Python ${PSP_PYTHON_VERSION} EXACT REQUIRED COMPONENTS Interpreter Development)
 
             # Run with exact version so its cached for pybind
-            find_package(PythonInterp ${PSP_PYTHON_VERSION} EXACT REQUIRED)
-            find_package(PythonLibs ${PSP_PYTHON_VERSION} EXACT REQUIRED)
+            # find_package(PythonInterp ${PSP_PYTHON_VERSION} EXACT REQUIRED)
+            # find_package(PythonLibs ${PSP_PYTHON_VERSION} EXACT REQUIRED)
 
             link_directories(${Python_LIBRARY_DIRS})
         endif()
-- 
2.40.0

