From e1a6b368b656e922e719c37044051d01ced90730 Mon Sep 17 00:00:00 2001
From: Tim Paine <3105306+timkpaine@users.noreply.github.com>
Date: Wed, 5 Apr 2023 11:09:32 -0400
Subject: [PATCH] pybind_cmake

---
 dist/cmake/modules/FindNumPy.cmake  | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/dist/cmake/modules/FindNumPy.cmake b/dist/cmake/modules/FindNumPy.cmake
index 4129be077..14ad03cdb 100644
--- a/dist/cmake/modules/FindNumPy.cmake
+++ b/dist/cmake/modules/FindNumPy.cmake
@@ -5,7 +5,7 @@

 cmake_minimum_required(VERSION 3.7.2)

-find_package( PythonInterp ${PSP_PYTHON_VERSION} EXACT REQUIRED )
+find_package( Python ${PSP_PYTHON_VERSION} EXACT REQUIRED )

 # Find out the include path
 execute_process(
--
2.40.0
