From 7209e096d803112c840e5f031c93e32cb2183510 Mon Sep 17 00:00:00 2001
From: Tim Paine <3105306+timkpaine@users.noreply.github.com>
Date: Fri, 21 Jul 2023 16:00:38 -0400
Subject: [PATCH] use boost shared

---
 dist/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dist/CMakeLists.txt b/dist/CMakeLists.txt
index 6a7af9e6c..badf261d2 100644
--- a/dist/CMakeLists.txt
+++ b/dist/CMakeLists.txt
@@ -294,7 +294,6 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
         set(CMAKE_FIND_ROOT_PATH "${CMAKE_FIND_ROOT_PATH};/usr/local/")
         find_package(Boost REQUIRED)
     else()
-        set(Boost_USE_STATIC_LIBS ON)
         find_package(Boost REQUIRED COMPONENTS system thread)
     endif()
 
-- 
2.41.0

