diff --git a/cpp/perspective/CMakeLists.txt b/cpp/perspective/CMakeLists.txt
index b06707506..3a7c1307b 100644
--- a/cpp/perspective/CMakeLists.txt
+++ b/cpp/perspective/CMakeLists.txt
@@ -300,7 +300,7 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
         set(CMAKE_FIND_ROOT_PATH "${CMAKE_FIND_ROOT_PATH};/usr/local/")
         find_package(Boost REQUIRED)
     elseif(NOT WIN32)
-        find_package(Boost REQUIRED COMPONENTS system)
+        find_package(Boost REQUIRED)
     endif()
 
     if(WIN32)
@@ -370,7 +370,7 @@ if(PSP_PYTHON_BUILD AND NOT PSP_PYODIDE)
     # Boost must be added after arrow to prevent linking bug on Windows.
     if(WIN32)
         set(Boost_USE_STATIC_LIBS ON)
-        find_package(Boost REQUIRED COMPONENTS system)
+        find_package(Boost REQUIRED)
     endif()
     if(NOT Boost_FOUND)
         message(FATAL_ERROR "${Red}Boost could not be located${ColorReset}")
@@ -378,7 +378,7 @@ if(PSP_PYTHON_BUILD AND NOT PSP_PYODIDE)
         psp_build_message("${Cyan}Found Boost: `Boost_INCLUDE_DIRS`: ${Boost_INCLUDE_DIRS}, `Boost_LIBRARY_DIRS` - ${Boost_LIBRARY_DIRS} ${ColorReset}")
         if(WIN32)
             include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
-            add_definitions(-DBOOST_UUID_FORCE_AUTO_LINK)
+            # add_definitions(-DBOOST_UUID_FORCE_AUTO_LINK)
         endif()
     endif()
 endif()
-- 
2.46.1

