diff --git a/rust/perspective-client/Cargo.toml b/rust/perspective-client/Cargo.toml
index ab3a89ee7..917d19c6b 100644
--- a/rust/perspective-client/Cargo.toml
+++ b/rust/perspective-client/Cargo.toml
@@ -35,7 +35,7 @@ rustdoc-args = ["--html-in-header", "docs/index.html"]

 [features]
 default = []
-external-proto = ["protobuf-src"]
+external-proto = []
 external-protoc = []

 [lib]
@@ -44,7 +44,7 @@ path = "src/rust/lib.rs"

 [build-dependencies]
 prost-build = { version = "0.12.3" }
-protobuf-src = { version = "2.0.1", optional = true }
+# protobuf-src = { version = "2.0.1", optional = true }

 [dependencies]
 async-lock = { version = "2.5.0" }
diff --git a/rust/perspective-client/build.rs b/rust/perspective-client/build.rs
index adf3ae794..24c374cc7 100644
--- a/rust/perspective-client/build.rs
+++ b/rust/perspective-client/build.rs
@@ -36,8 +36,8 @@ fn prost_build() -> Result<()> {

         println!("cargo:rerun-if-changed={}", proto_file.to_str().unwrap());

-        #[cfg(feature = "external-proto")]
-        std::env::set_var("PROTOC", protobuf_src::protoc());
+        // #[cfg(feature = "external-proto")]
+        // std::env::set_var("PROTOC", protobuf_src::protoc());

         prost_build::Config::new()
             // .bytes(["ViewToArrowResp.arrow", "from_arrow"])
