diff --git a/rust/perspective-python/Cargo.toml b/rust/perspective-python/Cargo.toml
index 4d9572b39..6351c70ea 100644
--- a/rust/perspective-python/Cargo.toml
+++ b/rust/perspective-python/Cargo.toml
@@ -58,11 +58,7 @@ perspective-server = { version = "3.1.0", path = "../perspective-server" }
 pollster = "0.3.0"
 extend = "1.1.2"
 futures = "0.3.28"
-pyo3 = { version = "0.21.2", features = [
-    "extension-module",
-    "serde",
-    "abi3-py39",
-] }
+pyo3 = { version = "0.21.2", features = ["extension-module", "serde"] }
 pythonize = "0.21.1"
 tracing = { version = ">=0.1.36" }
 tracing-subscriber = { version = "0.3.15", features = ["env-filter"] }
