diff --git a/rust/perspective-python/build.mjs b/rust/perspective-python/build.mjs
index 78fbd5083..d8deb97c1 100644
--- a/rust/perspective-python/build.mjs
+++ b/rust/perspective-python/build.mjs
@@ -75,7 +75,7 @@ if (build_wheel) {
         process.platform === "linux"
     ) {
         target =
-            "--target=x86_64-unknown-linux-gnu --compatibility manylinux_2_28";
+            "--target=x86_64-unknown-linux-gnu --compatibility linux";
     } else if (
         process.env.PSP_ARCH === "aarch64" &&
         process.platform === "linux"
@@ -87,6 +87,12 @@ if (build_wheel) {
         flags += " -vv";
     }

+    if (process.env.CONDA_BUILD && process.env.PYTHON) {
+        fs.accessSync(process.env.PYTHON, fs.constants.F_OK | fs.constants.X_OK);
+	// FIXME: shell-quote the string, or build up command line flags as an array instead
+	flags += ` --interpreter=${process.env.PYTHON}`;
+    }
+
     cmd.sh(`maturin build ${flags} --features=external-cpp ${target}`);
 }

