diff --git a/cpp/perspective/CMakeLists.txt b/cpp/perspective/CMakeLists.txt
index 29f0be177..f79514cd9 100644
--- a/cpp/perspective/CMakeLists.txt
+++ b/cpp/perspective/CMakeLists.txt
@@ -364,6 +364,8 @@ psp_build_dep("flatbuffers" "${PSP_CMAKE_MODULE_PATH}/flatbuffers.txt.in")
 psp_build_dep("lz4" "${PSP_CMAKE_MODULE_PATH}/lz4.txt.in")

 # Build minimal arrow itself
+# Arrow detects conda and looks on system for sources - perspective needs it to use bundled sources
+set(ARROW_DEPENDENCY_SOURCE "BUNDLED" CACHE STRING "forgive me" FORCE)
 psp_build_dep("arrow" "${PSP_CMAKE_MODULE_PATH}/arrow.txt.in")

 if(PSP_PYTHON_BUILD AND NOT PSP_PYODIDE)
