diff --git a/cmake/arrow/CMakeLists.txt b/cmake/arrow/CMakeLists.txt
index b4ebab453..a60d4cbe0 100644
--- a/cmake/arrow/CMakeLists.txt
+++ b/cmake/arrow/CMakeLists.txt
@@ -2,20 +2,21 @@ include(CheckCCompilerFlag)
 
 set(CMAKE_SHARED_LIBRARY_SUFFIX .so)
 
-if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-    check_c_compiler_flag("-arch x86_64" x86_64Supported)
-    check_c_compiler_flag("-arch arm64" arm64Supported)
-
-    if(x86_64Supported AND arm64Supported)
-        set(CMAKE_OSX_ARCHITECTURES "x86_64;arm64" CACHE STRING "Build universal architecture for OSX" FORCE)
-    elseif(x86_64Supported)
-        set(CMAKE_REQUIRED_LINK_OPTIONS "-arch;x86_64")
-        set(CMAKE_OSX_ARCHITECTURES "x86_64" CACHE STRING "Build universal architecture for OSX" FORCE)
-    elseif(arm64Supported)
-        set(CMAKE_REQUIRED_LINK_OPTIONS "-arch;arm64")
-        set(CMAKE_OSX_ARCHITECTURES "arm64" CACHE STRING "Build universal architecture for OSX" FORCE)
-    endif()
-endif()
+# TODO This now needs to be set externally because we cross-compile.
+# if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+#     check_c_compiler_flag("-arch x86_64" x86_64Supported)
+#     check_c_compiler_flag("-arch arm64" arm64Supported)
+#
+#     if(x86_64Supported AND arm64Supported)
+#         set(CMAKE_OSX_ARCHITECTURES "x86_64;arm64" CACHE STRING "Build universal architecture for OSX" FORCE)
+#     elseif(x86_64Supported)
+#         set(CMAKE_REQUIRED_LINK_OPTIONS "-arch;x86_64")
+#         set(CMAKE_OSX_ARCHITECTURES "x86_64" CACHE STRING "Build universal architecture for OSX" FORCE)
+#     elseif(arm64Supported)
+#         set(CMAKE_REQUIRED_LINK_OPTIONS "-arch;arm64")
+#         set(CMAKE_OSX_ARCHITECTURES "arm64" CACHE STRING "Build universal architecture for OSX" FORCE)
+#     endif()
+# endif()
 
 set(ARROW_SRCS
 
