diff --git a/rust/perspective-server/build/psp.rs b/rust/perspective-server/build/psp.rs
index 92e8fd506..62dc70f8a 100644
--- a/rust/perspective-server/build/psp.rs
+++ b/rust/perspective-server/build/psp.rs
@@ -62,7 +62,7 @@ pub fn cmake_build() -> Result<Option<PathBuf>, std::io::Error> {
     dst.define("CMAKE_BUILD_TYPE", profile.as_str());
     if std::env::var("PSP_ARCH").as_deref() == Ok("x86_64") {
         dst.define("CMAKE_OSX_ARCHITECTURES", "x86_64");
-    } else if std::env::var("PSP_ARCH").as_deref() == Ok("arm64") {
+    } else if std::env::var("PSP_ARCH").as_deref() == Ok("aarch64") {
         dst.define("CMAKE_OSX_ARCHITECTURES", "arm64");
     }

