diff --git a/cpp/perspective/CMakeLists.txt b/cpp/perspective/CMakeLists.txt
index 4abf7ab86..fca7977cd 100644
--- a/cpp/perspective/CMakeLists.txt
+++ b/cpp/perspective/CMakeLists.txt
@@ -326,7 +326,6 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
         set(CMAKE_FIND_ROOT_PATH "${CMAKE_FIND_ROOT_PATH};/usr/local/")
         find_package(Boost REQUIRED)
     else()
-        set(Boost_USE_STATIC_LIBS ON)
         find_package(Boost REQUIRED COMPONENTS system)
     endif()
 
