From 1c6231efe712bdd8db0519e48ba4d1cb4386374c Mon Sep 17 00:00:00 2001
From: Tom Jakubowski <tom@crystae.net>
Date: Sun, 29 Sep 2024 18:54:10 +0000
Subject: [PATCH] ../perspective-feedstock/recipe/patches/shared_boost.diff

---
 cpp/perspective/CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cpp/perspective/CMakeLists.txt b/cpp/perspective/CMakeLists.txt
index f2790cf76..b06707506 100644
--- a/cpp/perspective/CMakeLists.txt
+++ b/cpp/perspective/CMakeLists.txt
@@ -299,8 +299,7 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
     if(PSP_PYODIDE)
         set(CMAKE_FIND_ROOT_PATH "${CMAKE_FIND_ROOT_PATH};/usr/local/")
         find_package(Boost REQUIRED)
     elseif(NOT WIN32)
-        set(Boost_USE_STATIC_LIBS ON)
         find_package(Boost REQUIRED COMPONENTS system)
     endif()
 
-- 
2.46.1

